describe/ProtNs.spec.ts

```diff
@@ -0,0 +1,52 @@
+import { expect } from "chai";
+import { describeProtNs } from "./ProtNs";
+
+const scl = new DOMParser().parseFromString(
+  `<SCL xmlns="http://www.iec.ch/61850/2003/SCL">
+    <DataTypeTemplates>
+        <DAType id="someDAType">
+            <ProtNs>IEC 61850-8-1:2003</ProtNs>
+        </DAType>
+        <DAType id="someDifferentDAType1">
+            <ProtNs>IEC 61850-8-1:2007</ProtNs>
+        </DAType>
+        <DAType id="someDifferentDAType2">
+            <ProtNs></ProtNs>
+        </DAType>
+        <DAType id="someOtherDAType">
+            <ProtNs type="8-MMS">IEC 61850-8-1:2003</ProtNs>
+        </DAType>
+    </DataTypeTemplates>
+  </SCL>`,
+  "application/xml"
+);
+
+const baseProtNs = scl.querySelector('DAType[id="someDAType"] > ProtNs')!;
+const diffProtNs1 = scl.querySelector(
+  'DAType[id="someDifferentDAType1"] > ProtNs'
+)!;
+const diffProtNs2 = scl.querySelector(
+  'DAType[id="someDifferentDAType2"] > ProtNs'
+)!;
+const equalProtNs = scl.querySelector('DAType[id="someOtherDAType"] > ProtNs')!;
+
+describe("Describes the SCL element Val", () => {
```

Comment:
Same as for all the other tests, I suggest `describe("describeProtNs(element)", ...`